customize.dist/translations/messages.es.js

```diff
@@ -18,7 +18,7 @@ define(function () {
     out.synchronizing = "Sincronización";
     out.reconnecting = "Reconectando...";
     out.lag = "Retraso";
-    out.readonly = 'Solo lectura';
+    out.readonly = 'Sólo lectura';
```

Comment:
Do other variants of Spanish have their own accentuation rules? I thought they would follow what the Spain variant does.